test/JDBC/input/alter_authorization_change_db_owner-vu-verify.mix

```diff
@@ -0,0 +1,407 @@
+-- tsql
+create login dba_login with password='12345678'
+go
+create login new_owner_login with password='12345678'
+go
+create login new_owner_login2 with password='12345678'
+go
+alter server role sysadmin add member dba_login
+go
+-- end preparation
+
+-- tsql user=dba_login password=12345678
+select suser_name(), db_name()
+go
+create database change_owner_db
+go
+alter authorization on database::no_such_db_chgownerdb to new_owner_login

```

Comment:
would really like to see some case-insensitve test cases

Reply:
see lower down in the file